rag_experiment_accelerator/config/config.py

```diff
@@ -74,7 +77,10 @@ def __init__(
         self.OVERLAP_SIZES = config_json["chunking"]["overlap_size"]
         self.EF_CONSTRUCTIONS = config_json["ef_construction"]
         self.EF_SEARCHES = config_json["ef_search"]
-        self.NAME_PREFIX = config_json["name_prefix"]
+        self.INDEX_NAME_PREFIX = config_json["index_name_prefix"]
+        self.EXPERIMENT_NAME = config_json["experiment_name"] or self.INDEX_NAME_PREFIX
+        self.JOB_NAME = config_json["job_name"]
```

Comment:
I think either add suffix for each job name according to the current `index_config` combination, or simply add incremental number.
more other all mlflow tags which are not computational show move to the first step from the last step (eval).